src/index.js

```diff
@@ -49,4 +30,62 @@ const listFooter = `</ul>
   <button type="button">Clear all completed</button>
 </div>`;
 
-document.querySelector('.content-1-1').innerHTML = listHead + data.map((item) => listItem(item)).join('') + listFooter;
+const display = () => {
+  document.querySelector('.content-1-1').innerHTML = listHead + data.map((item) => listItem(item)).join('') + listFooter;
+
+  document.querySelectorAll('.value').forEach((input, i) => {
+    input.onclick = (e) => {
+      const activeElement = document.querySelector('.active');
+
+      if (activeElement) {
+        const icons = [...activeElement.childNodes].filter((icon) => icon.innerHTML === 'more_vert' || icon.innerHTML === 'delete');
+
+        icons[0].classList.remove('hide');
+        icons[1].classList.add('hide');
+
+        activeElement.classList.remove('active');
+      }
+      e.target.parentElement.parentElement.classList.add('active');
+      const icons = [...e.target.parentElement.parentElement.childNodes].filter((icon) => icon.innerHTML === 'more_vert' || icon.innerHTML === 'delete');
+      icons[0].classList.add('hide');
+      icons[1].classList.remove('hide');
+
+      icons[1].onclick = () => {
+        if (!input.value) {
+          data = removeTask(i, data);
+
+          display();
+        }
+      };
+
+      e.target.onkeypress = (event) => {
```

Comment:
- [ ] [ OPTIONAL] Good job 👏  by implementing the deleting function as on the video shared above ⬆️  but I would like you to consider the fact that we can delete the task when the delete icon is  checked As mentioned by the previous reviewer 👍